### ISSUE TITLE
Suggestions for edits to README

### ISSUE THREAD

**Issue by hakabuk** [AurelienLourot/ghuser.io#28]
While reading the [README](https://github.com/AurelienLourot/ghuser.io/blob/master/README.md) I had some questions:

- Having a general link in the first bullet point was a bit disappointing. I was curious to see the profile! At the least we want to add a real link to a profile next to the general 404 link, so the user can really see a profile example. 
- Should be `UI/layout as close as possible *to* the official profiles.`
- Regardless of the typo in the previous bullet point, this should be either edited or removed. If it's so similar to github's profiles - why should I want it? :) It can be also just lowered down the list, IMO. 
- I feel bullet points 7,8,9 (start counting from 1) should be higher in the list. I might be wrong, but don't these offer something github doesn't offer? If I'm assuming wrong - ignore this one :) 

In conclusion, my feeling is this repo aims to give me something Github can't, so rather than point out how similar it is to Github - maybe first point out and show where it stands out? Please correct me if my assumptions are wrong. 

I hope these help/ make sense (p.s, I love the minimalist design and the overall look and UI of the profiles!).

**Comment by AurelienLourot** [comment]
Thank you so much for the feedback! I agree with everything, let's fix it :)

Some useful info on top of what you said:

* See #5 for improving the 404 page
* actually the points 3-10 (not only 7-9) are something github can't do, and yes we should make this more clear:

  * (3) github doesn't show the repos that you don't own and have contributed to 1 year ago
  * (4) the repos aren't well sorted on github
  * (5) you don't gain stars by having contributed to a project on github. only projects get stars, not user
  * (6) see 5
  * (10) on github, organizations have logos but not repos

**Comment by AurelienLourot** [comment]
By the way the reason why I wanted to stay close to GitHub's UI is because I found somewhat similar projects (they were trying to build some kind of "github resume") but by having a different UI it didn't feel serious somehow. Not sure why, it's more emotional than rational, but by following GitHub's UI the displayed data *feels* more trustable.

**Comment by AurelienLourot** [comment]
I would also mention one more thing that we do and github doesn't: we make easily visible how much (in %) the user has contributed to a project. (I'm working on this right now in #1)
